app/components/learnergroup-members.js

```diff
@@ -1,7 +1,19 @@
 import Ember from 'ember';
 import DS from 'ember-data';
+import InPlace from 'ilios/mixins/inplace';
+
+const { computed, isEmpty, ObjectProxy, observer, RSVP } = Ember;
+const { all } = RSVP;
+
+export default Ember.Component.extend(InPlace, {
+  init() {
```

Comment:
Is there an advantage to doing this in an init() as opposed to just setting these as properties on the component?